systems/plants/test/CMakeLists.txt

```diff
@@ -6,6 +6,9 @@ if (eigen3_FOUND)
   add_executable(urdfKinTest urdfKinTest.cpp)
   include_directories( .. )
   target_link_libraries(urdfKinTest drakeRBM)
+  add_executable(rbmConstructorDestructorTest rbmConstructorDestructorTest.cpp)
+  include_directories( .. )
+  target_link_libraries(rbmConstructorDestructorTest drakeRBM)
```

Comment:
I didn't add_test it because the intended use case is with valgrind. Should I zap it then?